CHANGES.md

```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.364
+  - Add a new resource to OSL clusters/cluster_log
```

Comment:
Usually we write down "Update model version v0.0.3xx" and then the details under. See v0.1.361 for example.

Reply:
done